src/resources/views/layout.blade.php

```diff
@@ -44,7 +44,7 @@
     <script src="https://oss.maxcdn.com/respond/1.4.2/respond.min.js"></script>
     <![endif]-->
 </head>
-<body class="hold-transition {{ config('backpack.base.skin') }} sidebar-mini">
+<body class="hold-transition {{ config('backpack.base.skin') }} sidebar-mini {{ config('backpack.base.layout', '') }}">
```

Comment:
Hmm I see there are _two_ configs being thrown out on the same line, in the same ```class``` attribute here. So I'm wondering - couldn't people achieve the same thing by changing the ```skin``` to ```skin-purple fixed```?

If so, maybe a better solution would be to:
- (A) do nothing, let them use ```skin``` instead of ```layout```
- (B) rename the ```skin``` config to ```body_classes``` and have it default to ```hold-transition skin-purple sidebar-mini```

What do you think @tswonke ?

Reply:
Lol, you're right... it seems that I was a blockhead 😂 

`body_classes` might be a good idea, but I wouldn't change it (for now) - maybe in 4.0 with some other customization configs...